grade-c.json

```diff
@@ -0,0 +1,20 @@
+{
+	"plugins": [
+		"stylelint-no-unsupported-browser-features"
+	],
+	"rules": {
+		"plugin/no-unsupported-browser-features": [ true, {
+			"browsers": [
+				"chrome >= 1",
+				"firefox >= 3",
```

Comment:
Aren't we about to bump this to 3.6?

Reply:
And probably many other things because we don't support TLS 1.2. We can change it when we change it.

Reply:
Fair.